setup-service.sh

```diff
@@ -64,38 +64,38 @@ $SUDO cp -v $MYDIR/service/share/dbus.conf /usr/share/dbus-1/agama.conf
 #   (this could be left out but then we would rely
 #    on the manual startup via bin/agamactl)
 (
-  cd $MYDIR/service/share
-  DBUSDIR=/usr/share/dbus-1/agama-services
-
-  # cleanup previous installation
-  [[ -d $DBUSDIR ]] && $SUDO rm -r $DBUSDIR
-
-  # create services 
-  $SUDO mkdir -p $DBUSDIR
-  for SVC in org.opensuse.Agama*.service; do
-    sudosed "s@\(Exec\)=/usr/bin/@\1=$MYDIR/service/bin/@" $SVC $DBUSDIR/$SVC
-  done
-  sudosed "s@\(ExecStart\)=/usr/bin/@\1=$MYDIR/service/bin/@" \
-          systemd.service /usr/lib/systemd/system/agama.service
+	cd $MYDIR/service/share
+	DBUSDIR=/usr/share/dbus-1/agama-services
+
+	# cleanup previous installation
+	[[ -d $DBUSDIR ]] && $SUDO rm -r $DBUSDIR
+
+	# create services
+	$SUDO mkdir -p $DBUSDIR
+	for SVC in org.opensuse.Agama*.service; do
+		sudosed "s@\(Exec\)=/usr/bin/@\1=$MYDIR/service/bin/@" $SVC $DBUSDIR/$SVC
+	done
+	sudosed "s@\(ExecStart\)=/usr/bin/@\1=$MYDIR/service/bin/@" \
+		agama.service /usr/lib/systemd/system/agama.service
```

Comment:
You have changed the indentation of the whole file to change this line. I am not sure if that's correct at all.

Reply:
Ah, we have `.editorconfig` in `service` and `web` but not here at the top level :thinking: